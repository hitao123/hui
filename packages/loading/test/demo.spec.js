import { shallowMount } from '@vue/test-utils';
import { renderToString } from '@vue/server-test-utils';
import Loading from '../index.vue';
import LoadingDemo from '../demo/index.vue';

describe('Loading component', () => {
  it('should Loading as Circular', () => {
    const warpper = shallowMount(Loading, {
      propsData: {
        type: 'spinner',
        color: 'white'
      }
    });
    expect(warpper.html()).toMatchSnapshot();
  });

  it('should Loading as Spinner', () => {
    expect(renderToString(LoadingDemo)).toMatchSnapshot();
  });
});
