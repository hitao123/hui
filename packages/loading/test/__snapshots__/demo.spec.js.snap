// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div>
    <h-loading></h-loading>
    <h-loading color="white"></h-loading>
  </div>
  <div>
    <h-loading type="spinner"></h-loading>
    <h-loading type="spinner" color="white"></h-loading>
  </div>
</div>
`;
